edx_rest_framework_extensions/permissions/oauth_scopes.py

```diff
@@ -1,47 +1,29 @@
-""" Permission classes. """
+"""
+Permission classes to be used only for Views that need to support OAuth Scopes and/or OAuth
+Restricted Applications. See the following for understanding Restricted Applications and
+OAuth Scope enforcement in the platform:
+https://github.com/edx/edx-platform/tree/master/openedx/core/djangoapps/oauth_dispatch/docs/decisions
```

Comment:
I think a better link might be:
https://open-edx-proposals.readthedocs.io/en/latest/oep-0004-arch-oauth-scopes.html#scope-and-filter-implementation

Reply:
That link doesn't explain what RestrictedApplications are.